docs/designers-developers/developers/tutorials/javascript/versions-and-building.md

```diff
@@ -0,0 +1,14 @@
+
+## JavaScript versions and build step
```

Comment:
This section is important but feels disconnected and left me wondered _how one should go about the build step?_. I know Gutenberg is agnostic about what tools third-party authors use, but I still feel this doc should direct them to the best resources available at the moment. It can be changed later if/when Gutenberg itself provides this. I'd include here a reference to the [Create Gutenberg Block](https://github.com/ahmadawais/create-guten-block) project.

Reply:
yeh, this was tricky. I wanted to explain what they might see in the documentation but try to keep it simple.  It is probably in the wrong spot in my table of contents, maybe put it after troubelshooting and expand.

I'm a big fan of getting the person to quick wins as they go through a tutorial.
Do this. Run. See something.
Now add this. Run. See new something.
-> Repeat.

So including the extra info in the middle confuses it. 
I think I'll reorganize it to the end and can expand.